arangodb-net-standard.Test/GraphApi/GraphApiClientTest.cs

```diff
@@ -326,5 +326,79 @@ await _client.PostVertexCollectionAsync(graphName, new PostVertexCollectionBody(
             Assert.Equal(HttpStatusCode.NotFound, apiError.Code);
             Assert.Equal(1924, apiError.ErrorNum); // ERROR_GRAPH_NOT_FOUND
         }
+
+        [Fact]
+        public async Task DeleteVertexCollectionAsync_ShouldSucceed()
+        {
+            // Create a new graph
+
+            string graphName = nameof(DeleteVertexCollectionAsync_ShouldSucceed);
+
+            PostGraphResponse createResponse = await _client.PostGraphAsync(
+                new PostGraphBody()
+                {
+                    Name = graphName
+                });
+
+            // Add a vertex collection
+
+            string clxToAdd = nameof(DeleteVertexCollectionAsync_ShouldSucceed);
+
+            PostVertexCollectionResponse createVertexResponse = await _client.PostVertexCollectionAsync(
+                graphName,
+                new PostVertexCollectionBody()
+                {
+                    Collection = clxToAdd
+                });
+
+            var response = await _client.DeleteVertexCollectionAsync(graphName, clxToAdd, new DeleteVertexCollectionQuery
+            {
+                DropCollection = false
+            });
+
+            Assert.Equal(HttpStatusCode.Accepted, response.Code);
+            Assert.False(response.Error);
+
+            GetGraphsGraph graph = response.Graph;
+        }
+
+        [Fact]
+        public async Task DeleteVertexCollectionAsync_ShouldThrow_WhenGraphIsNotFound()
+        {
+            string graphName = nameof(DeleteVertexCollectionAsync_ShouldThrow_WhenGraphIsNotFound);
+            string clxToAdd = nameof(DeleteVertexCollectionAsync_ShouldThrow_WhenGraphIsNotFound);
```

Comment:
Maybe rename `clxToAdd`? I think this is the vertex collection to remove from the graph. Same below.